### PR TITLE
add rart75m keyboard and support via

### PR DESCRIPTION
**Pull request by alabahuy**
## Description

add new keyboard, rart75m. and support via

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — APPROVED** [review]

File: keyboards/rart/rart75m/info.json [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+                {"x":0, "y":5.25},
+                {"x":1.5, "y":5.25, "w":1.25},
+                {"x":2.75, "y":5.25, "w":1.25},
+                {"x":4, "y":5.25, "w":1.25},
+                {"x":5.25, "y":5.25, "w":6.25},
+                {"x":11.5, "y":5.25, "w":1.5},
+                {"x":13, "y":5.25, "w":1.5},
+                {"x":14.75, "y":5.5 },
+                {"x":15.75, "y":5.5 },
+                {"x":16.75, "y":5.5 }
```
Comment (fauxpark) [inline keyboards/rart/rart75m/info.json:97]:
```suggestion
                {"x":16.75, "y":5.5 }
```

File: keyboards/rart/rart75m/info.json [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+                {"x":5, "y":0},
+                {"x":6, "y":0},
+                {"x":7.25, "y":0},
+                {"x":8.25, "y":0},
+                {"x":9.25, "y":0},
+                {"x":10.25, "y":0},
+                {"x":11.5, "y":0},
+                {"x":12.5, "y":0},
+                {"x":13.5, "y":0},
+                {"x":14.5, "y":0},
```
Comment (fauxpark) [inline keyboards/rart/rart75m/info.json:22]:
```suggestion
                {"x":14.5, "y":0},
```

File: keyboards/rart/rart75m/info.json [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+                {"x":9, "y":2.25},
+                {"x":10, "y":2.25},
+                {"x":11, "y":2.25},
+                {"x":12, "y":2.25},
+                {"x":13, "y":2.25},
+                {"x":14, "y":2.25},
+                {"x":15, "y":2.25, "w":1.5},
+                {"x":17, "y":2.25},
+
+                {"x":0, "y":3.25},
```
Comment (fauxpark) [inline keyboards/rart/rart75m/info.json:56]:
```suggestion
                {"x":0, "y":3.25},
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/rart/rart75m/info.json (hunk 3 above)
Comment (alabahuy) [reply keyboards/rart/rart75m/info.json]: done @fauxpark :)

**Review by fauxpark — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**